.github/workflows/pkgcheck.yml

```diff
@@ -0,0 +1,28 @@
+name: CI pkgcheck
```

Comment:
Capitailize `PkgCheck` like the rest of the CI workflows..

Reply:
Done (though no intercaps, since the script pkgcheck.sh is all lower case, it'd be strange to capitalize the c)